starlite/openapi/controller.py

```diff
@@ -65,6 +66,35 @@ def get_schema_from_request(request: Request) -> "OpenAPI":
             raise ImproperlyConfiguredException("Starlite has not been instantiated with OpenAPIConfig")
         return request.app.openapi_schema
 
+    def should_serve_endpoint(self, request: Request) -> bool:
+        """This method verifies that the requested path is within the allowed
+        endpoints in the openapi_config.
+
+        Args:
+            request: A [Starlite][starlite.connection.Request] instance.
```

Comment:
```suggestion
            request: To be tested if endpoint activated.
```
Referencing the type of parameters in the description is unnecessary as the ref docs framework already renders that information with the arg description.

![image](https://user-images.githubusercontent.com/20659309/187309853-f92d1b52-1a12-4638-895e-c7f734f8e50f.png)